website/docs/docs/deploy/merge-jobs.md

```diff
@@ -13,27 +13,55 @@ By using CD in dbt Cloud, you can take advantage of deferral to build only the e
+    - **Compare changes against** &mdash; By default, it's set to compare changes against the environment you created the job from. This option allows dbt Cloud to check the state of the code in the PR against the code running in the deferred environment, so as to only check the modified code, instead of building the full table or the entire DAG. To change the default settings, you can select **No deferral** or choose a different environment. 
 1. (optional) Options in the **Advanced settings** section: 
     - **Environment variables** &mdash; Define [environment variables](/docs/build/environment-variables) to customize the behavior of your project when this job runs.
-    - **Target name** &mdash; Define the [target name](/docs/build/custom-target-names). Similar to **Environment Variables**, this option lets you customize the behavior of the project. 
+    - **Target name** &mdash; Define the [target name](/docs/build/custom-target-names). Similar to environment variables, this option lets you customize the behavior of the project. 
     - **Run timeout** &mdash; Cancel this job if the run time exceeds the timeout value.
     - **dbt version** &mdash; By default, it’s set to inherit the [dbt version](/docs/dbt-versions/core) from the environment. dbt Labs strongly recommends that you don't change the default setting. This option to change the version at the job level is useful only when you upgrade a project to the next dbt version; otherwise, mismatched versions between the environment and job can lead to confusing behavior.
     - **Threads** &mdash; By default, it’s set to 4 [threads](/docs/core/connect-data-platform/connection-profiles#understanding-threads). Increase the thread count to increase model execution concurrency.
-    - **Run source freshness** &mdash; Enable this option to invoke the `dbt source freshness` command before running this job. Refer to [Source freshness](/docs/deploy/source-freshness) for more details.
 
-## Example 
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/example-create-merge-job.png" title="Example of creating a merge job"/>
 
-<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/example-create-merge-job.png" width="90%" title="Example of creating a merge job"/>
\ No newline at end of file
+## Verify push events in Git
+
+Merge jobs require push events so make sure they've been enabled in your Git provider, especially if you have an already-existing Git integration. However, for a new integration setup, you can skip this check since push events are typically enabled by default. 
+
+<Expandable alt_header="GitHub example" >
+
+The following is a GitHub example of when the push events are already set: 
```

Comment:
Would be nice to have details on how you wind up on this page. I actually just tried to go to find this in the GitHub settings to say something like "maybe let's provide a link to the page like `https://github.com/dbt-labs/docs.getdbt.com/settings/XXXX`" but i couldn’t find where it lived, which is sorta the point 😅 

ditto for the other providers too.

Reply:
per [slack convo](https://dbt-labs.slack.com/archives/C079137E39P/p1719420832303379), will keep as is. we can follow up later if needed

Reply:
That Slack convo is not loading for me. Can you provide some context for why we're not providing this info? I'm in agreement with @joellabes that if we're going to show the screenshots of the location, we should at least pop in a mention of where it's located in their environment. We don't necessarily have to walk them through it, but say something like "the XXXXX settings page in GitHub/GitLab/etc